modules/ravestate_telegramio/telegram_bot.py

```diff
@@ -69,7 +118,7 @@ def telegram_output(ctx: ContextWrapper):
     # TODO don't instantiate the updater every time
     token = ctx.conf(key=TOKEN_CONFIG_KEY)
     if not token:
-        logging.error('telegram-token is not set. Shutting down telegramio')
+        logger.error('telegram-token is not set. Shutting down telegramio')
```

Comment:
Would export private method here since we do the same thing in 97-100 and 118-121

Reply:
@Toseban maybe move creation of telegram output state into the startup state and refer to the token via closure - this would also fix the `# TODO don't instantiate the updater every time`

Reply:
See #38.